### ISSUE TITLE
Segfault in video_begin

### ISSUE THREAD

**Issue by dkmoz** [uTox/uTox#951]
```
uToxVideo:    video init done!
uToxVideo:    video init done!
uToxVideo:    started video

Thread 1 "utox" received signal SIGSEGV, Segmentation fault.
0x0000000000446299 in video_begin (id=65535, name=0x5f5600 "Video Preview", name_length=13, width=426, height=262) at utox/src/xlib/video.c:67
67    if (*win) {
(gdb) bt
#0  0x0000000000446299 in video_begin (id=65535, name=0x5f5600 "Video Preview", name_length=13, width=426, height=262) at utox/src/xlib/video.c:67
#1  0x000000000047ba50 in utox_message_dispatch (utox_msg_id=AV_VIDEO_FRAME, param1=65535, param2=1, data=0x603000053350) at utox/src/utox.c:544
#2  0x000000000046efbc in doevent (event=...) at utox/src/xlib/event.c:665
#3  0x000000000043e6fa in main (argc=2, argv=0x7fffffffe8d8) at utox/src/xlib/main.c:816

```

1. select desktop from video input device
2. test video preview

I'm unable to reproduce this on PC but it segfaults every time on my laptop

**Comment by endoffile78** [comment]
Will be fixed in #909.

**Comment by endoffile78** [comment]
Fixed in #909
